app/routines/update_user_salesforce_info.rb

```diff
@@ -191,6 +191,13 @@ def cache_contact_data_in_user!(contact, user)
 
       user.faculty_status = case contact.faculty_verified
       when "Confirmed"
+        SecurityLog.create!(
+            user: user.try(:is_anonymous?) ? nil : user,
+            application: nil,
+            remote_ip: nil,
+            event_type: :faculty_verified,
+            event_data: user
```

Comment:
This field should be a json object, not a ruby object. We don't usually pass in a user to event_data (except apparently in a few tests for some reason). I recommend you set the event_data to something like `{ user_id: user.id }`.